_posts/en/meetings/2018-06-28-meeting.md

```diff
@@ -0,0 +1,231 @@
+part of the user's wallet.
+
+## High priority for review
+
+**Background:** each meeting, Bitcoin Core developers discuss which Pull
+Requests (PRs) the meeting participants think most need review in the
+upcoming week.  Some of these PRs are related to code that contributors
+especially want to see in the next release; others are PRs that are
+blocking further work or which require significant maintenance (rebasing)
+to keep in a pending state.  Any capable reviewers are encouraged to
+visit the project's list of [current high-priority
+PRs][].
+
+**Discussion ([log][log hipri]):** the only PR specifically discussed
+was [#12196][], which adds a `scantxoutset` RPC to allow applications to
+search all current Unspent Transaction Outputs (UTXOs) for those that match a
+particular address, public key, private key, or HD wallet extended
+public key (xpub).
+
+Pieter Wuille suggested that the PR's author, Jonas Schnelli, remove
+xpub support from `scantxoutset` for now and Wuille would later PR an
+update to `scantxoutset` that adds support for the output script
+descriptors described later in these meeting notes.  If Wuille is
+delayed in doing so, Schnelli can just add the previously-written xpub
+support back in before the Bitcoin Core 0.17 feature freeze.  Schnelli
+agreed.
+
+*Note: extended discussion of output script descriptors during this part
+of the meeting has been moved to a [separate section](#output-script-descriptors).*
+
+## Cipherseed
+
+**Background:** (Part of the discussion.)
+
+**Discussion ([log][log cipherseed]):** Jonas Schnelli requested and
+introduced the topic, "I have a [specification draft][draft bip
+cipherseed] for a new seed format similar to BIP39 with some neat
+properties and---before sending [it] to the mailing list---would
+appreciate feedback."
+
+**Conclusion:** Schnelli said it was "more an announcement than a topic"
+and so discussion was deferred until people had a chance to read the
+draft.
+
+## Core's BIP32 derivation "standard"
+
+**Background:** the HD wallet specification, [BIP32][], defines how a
+set of private keys and public keys (and, therefore, addresses) can be
+derived from a 128-bit to 512-bit random seed.  
+
+**Discussion ([log][log core hd]):** Jonas Schnelli requested and
+introduced the topic, "It came up today in a discussion [that] Core's
+BIP32 derivation scheme is not specified in a BIP.  Some people think
+it's vanilla/native BIP32, but it's not, while other [wallets] do native
+BIP32."
+
+Wladimir van der Laan agreed "it would be good if the difference would
+be documented somewhere."  Luke Dashjr opposed it being a BIP and Pieter
+Wuille agreed, with Andrew Chow suggesting that "just documenting the
+derivation in the [documentation repository][docs repo] is sufficient."
+
+*Note: extended discussion of output script descriptors during this part
+of the meeting has been moved to a [separate section](#output-script-descriptors).*
+
+## P2Plink ephemeral encryption
```

Comment:
Space after P2P?

Reply:
Normally I would, but that was how it was set as `#topic` and i usually try to keep the summary section titles as close to the actual topics in the meetbot summary as possible to help people who are cross referencing them using searches.  Note that in this case I did make one adjustment, I added a trailing 'n' to the actual `#topic` which was

    #topic P2Plink ephemeral encryptio

I'm not insistent on keeping them consistent if the increase in clarity is worth the break in grepability, so let me know if you think that space is important.